src/middlewared/middlewared/alert/source/collectd.py

```diff
@@ -34,15 +49,15 @@ def check_sync(self):
         alerts = []
         for k, v in list(data.items()):
             if k == "ctl-ha/disk_octets":
-                title = "CTL HA link is actively used, check initiators connectivity"
+                text = "CTL HA link is actively used, check initiators connectivity."
```

Comment:
Not clear what this means.  The first part maybe means:
s/is actively used/is already in use/
Not sure what "check initiators connectivity" means.  Please expand--what type of connectivity?  How can the reader check this?  Maybe it should be a different form, like "CTL HA link already in use. Please verify that initiators can ..."
That ... in the last part could be "ping the VIP" or can resolve an IP address or something.  Don't know, not enough information.

Reply:
This alert means that some initiator(s) sends requests though the passive controller.  It means either initiator(s) does not support ALUA and just do round-robin, then user should just know it is sub-optimal, or initiator(s) lost connectivity to primary controller and this is a network failure condition.